pycbc/transforms.py

```diff
@@ -1492,13 +1509,30 @@ class Mass1Mass2ToMchirpQ(MchirpQToMass1Mass2):
     """
     name = "mass1_mass2_to_mchirp_q"
     inverse = MchirpQToMass1Mass2
-    _inputs = inverse._outputs
-    _outputs = inverse._inputs
     transform = inverse.inverse_transform
     inverse_transform = inverse.transform
     jacobian = inverse.inverse_jacobian
     inverse_jacobian = inverse.jacobian
 
+    def __init__(self, mass1_param=None, mass2_param=None, mchirp_param=None,
+                 q_param=None):
+        if mass1_param is None:
+            mass1_param = parameters.mass1
+        if mass2_param is None:
+            mass2_param = parameters.mass2
+        if mchirp_param is None:
+            mchirp_param = parameters.mchirp
+        if q_param is None:
+            q_param = parameters.q
+        self.mass1_param = mass1_param
+        self.mass2_param = mass2_param
+        self.mchirp_param = mchirp_param
+        self.q_param = q_param
+        self._inputs = [self.mass1_param, self.mass2_param]
+        self._outputs = [self.mchirp_param, self.q_param]
+        super(MchirpQToMass1Mass2, self).__init__()
```

Comment:
I think codacy is right that this should be `` super(Mass1Mass2ToMchirpQ, self).__init__()``.

Reply:
@cmbiwer Ok, I called the `BaseTransform` directly there.